src/sentry/api/event_search.py

```diff
@@ -2042,6 +2043,19 @@ def result_type_fn(function_arguments, parameter_values):
             transform="uniqIf(user, greater(duration, {tolerated:g}))",
             default_result_type="number",
         ),
+        Function(
+            "misery",
```

Comment:
nit: since we'll eventually rename this to user_misery, can we rename this to `user_misery.prototype` or `.new`?